src/character.cpp

```diff
@@ -2306,3 +2282,63 @@ std::string Character::extended_description() const
 
     return replace_colors( ss.str() );
 }
+
+template <typename Getter>
+float Character::calc_mutation_value( Getter getter ) const
```

Comment:
YAGNI, what evidence do you have that this requires hard-to-find lambda with caches all over the place?